tests/test_model.py

```diff
@@ -1653,3 +1654,32 @@ def test_model_logp_fast_compile():
 
     with pytensor.config.change_flags(mode="FAST_COMPILE"):
         assert m.point_logps() == {"a": -1.5}
+
+
+class TestModelGraphs:
+    @staticmethod
+    def school_model(J: int) -> pm.Model:
+        y = np.array([28, 8, -3, 7, -1, 1, 18, 12])
+        sigma = np.array([15, 10, 16, 11, 9, 11, 10, 18])
+        with pm.Model(coords={"school": np.arange(J)}) as schools:
+            eta = pm.Normal("eta", 0, 1, dims="school")
+            mu = pm.Normal("mu", 0, sigma=1e6)
+            tau = pm.HalfCauchy("tau", 25)
+            theta = mu + tau * eta
+            pm.Normal("obs", theta, sigma=sigma, observed=y, dims="school")
+        return schools
+
+    def test_graphviz(self) -> None:
+        model = self.school_model(J=8)
+        g_actual = model.graphviz()
```

Comment:
Technically, one could use patch/mock and only assert that everything was forwarded to the underlying `model_to_graphviz` function

Reply:
I added such mock in https://github.com/pymc-devs/pymc/pull/6865/commits/ec02a583a97c7b0fff1c74a6452e5d9ffc682df8 . Open for feedback regarding the tests we cactually need and which ones we can delete